backend/bufftracer/src/getClientFromHeader.ts

```diff
@@ -0,0 +1,12 @@
+import { GraphQlRequest, RPCRequest } from './types'
+
+export const HEADER_CLIENT_ID = 'x-buffer-client-id'
```

Comment:
more of a question, but in datadog will we be able to filter by both the client and the client version? We are currently sending the app version with the client header - just wondering if it will be possible to narrow things down in data dog. E.g "This endpoint hasn't been used since the android app version 6.0, which is from 5 years ago"

Reply:
@hitherejoe, the client here represents both the client+version. So, this is how they will appear in DD: https://share.buffer.com/o0uPWg82.

You can then filter by a specific one if you want in a single dropdown (instead of client + version): https://share.buffer.com/4gul6yO4. 

Screenshot got taken from there: https://app.datadoghq.com/dashboard/meb-fbz-gmw/buffertools-api-usage-metric?tpl_var_client=%2A&from_ts=1633242996919&to_ts=1633329396919&live=true

If you think it would be better to have two different fields, we can change that 😊

Reply:
perfect! Was just curious that's all - I think this will be great as it is for now 🙌